### DISCUSSION TITLE
Are there any intentions to create multi-threaded support in the future?

### DISCUSSION THREAD

**Discussion by NikolaGrgic** [DioxusLabs/dioxus#47] [General]
Is this something intended to be implemented in the future, preferably, near future?

**Comment by jkelleyrtp** [comment]
> Is this something intended to be implemented in the future, preferably, near future?

What would your use case be? Like the VirtualDom needs to be Send or Sync too?

In practice to make it Send, we would just need to swap out the Cells/Rcs for their threadsafe variants.
